app/src/main/res/layout/dialog_fragment_pickers.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
do you need a constraint layout surrounding the other layout?

Reply:
Not really. Removed